overlord/configstate/configcore/refresh.go

```diff
@@ -50,6 +50,17 @@ func validateRefreshSchedule(tr Conf) error {
 		}
 	}
 
+	refreshHoldOnMeteredStr, err := coreCfg(tr, "refresh.hold-on-metered")
```

Comment:
I suggest having this as

_refresh.metered = hold_

Besides being more explicit, it allows us to have more than two settings, and also gives us some freedom to decide what the default should be.

Reply:
Done